airdrop/job/reward_processors/nunet_reward_processor.py

```diff
@@ -28,7 +29,7 @@ def set_comment(self, comment):
         self._comment = comment
     
     def set_reward(self, reward):
-        self._reward = reward
+        self._reward = round(reward,0)
```

Comment:
Why are we doing this ?

Reply:
Cause of the issue we found a few days back. DApp cannot deal with decimal points and since we are using Python Decimal we always get decimal points